types/next-redux-saga/next-redux-saga-tests.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import withRedux from 'next-redux-wrapper';
+import withReduxSaga from 'next-redux-saga';
```

Comment:
This should be `import withReduxSaga = require('next-redux-saga')`, or you could add `"esModuleInterop": true` to `tsconfig.json` compiler options.